frontend/src/components/recipe_card/recipe_card.js

```diff
@@ -4,24 +4,50 @@ import './recipe_card.css';
 
 
 const Recipe = (props) => {
+
+  const mealDifficulty = () => {
+    if (props.difficulty === "EASY") {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/low-connection.png" alt="Easy Difficulty Rating"></img>
+      )
+    }
+    if (props.difficulty === "MEDIUM") {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/medium-connection.png" alt="Medium Difficulty Rating"></img>
+      )
+    }
+    if (props.difficulty === "HARD") {
```

Comment:
I think we may be calling this `Advanced` as opposed to `Hard`, which has a somewhat negative connotation.

Reply:
Done!